docs/upgrade-guide.md

```diff
@@ -2,23 +2,54 @@
 
 ## Upgrading to v9.0
 
-### Custom Layers
+### General changes
 
-Create models with device instead of gl.
+- `Deck.pickObjects()`  -  minor breaking change: if the same `data` is used by multiple top-level layers (e.g. a `ScatterplotLayer` and a `TextLayer`) that are visible in the picking bounds, `pickObjects` will yield one result for each picked object+layer combination, instead of one result for each picked object in previous versions.
```

Comment:
```suggestion
- `Deck.pickObjects()`  -  minor breaking change: if the same `data` is used by multiple top-level layers (e.g. a `ScatterplotLayer` and a `TextLayer`) that are visible in the picking bounds, `pickObjects` will yield one result for each picked object+layer combination, instead of one result for each picked object in previous versions.

### Typescript

Typescript is now enabled on all modules. The `'@deck.gl/xxx/typed'` packages are removed. 
```